src/rpdk/core/contract/resource_client.py

```diff
@@ -186,6 +187,21 @@ def assert_write_only_property_does_not_exist(self, resource_model):
             ), "The model MUST NOT return properties defined as \
                 writeOnlyProperties in the resource schema"
 
+    def get_metadata(self, metadata_key):
```

Comment:
if you want to reuse this method, then would be great not to hardcode `false` but use it as a arg with default (would be great for variation)

Reply:
valid point. Not sure we will be reusing this, I was thinking of hardcoding the metadat_key too